planner/core/plan_cacheable_checker.go

```diff
@@ -135,21 +135,6 @@ func (checker *cacheableChecker) Enter(in ast.Node) (out ast.Node, skipChildren
 				return in, true
 			}
 		}
-	case *ast.Limit:
```

Comment:
How about just commenting these lines out instead of removing them? Since we'll add a switch later.

Reply:
good idea